tsdb/index/postings.go

```diff
@@ -57,13 +59,15 @@ var ensureOrderBatchPool = sync.Pool{
 type MemPostings struct {
 	mtx     sync.RWMutex
 	m       map[string]map[string][]storage.SeriesRef
+	lvs     map[string][]string
 	ordered bool
 }
 
 // NewMemPostings returns a memPostings that's ready for reads and writes.
 func NewMemPostings() *MemPostings {
 	return &MemPostings{
 		m:       make(map[string]map[string][]storage.SeriesRef, 512),
+		lvs:     make(map[string][]string, 512),
```

Comment:
Given we have a constant for `2`, why not one for `512` ?

Reply:
Done in https://github.com/prometheus/prometheus/pull/15426/commits/c3e34b27fbca3b36f7537972f496324a8a25f98f